### DISCUSSION TITLE
Possible to use for this tag format?

### DISCUSSION THREAD

**Discussion by MiguelNdeCarvalho** [renovatebot/renovate#13550] [Request Help] [answered]
Hey,

Is it possible to configure Renovate to bump images with this tag format?

old tag: `3.0.6.1335-ls123`
new tag: `3.0.6.1342-ls131`

I have tried by using this `versioning: regex:^(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)\\.(?<build>\\d+)?(-ls(?<compatibility>\\d+))?$`

Thanks,
MiguelNdeCarvalho

**Comment by rarkins** [comment]
Is the lsxxx part important (e.g. needs to match), or can it be disregarded?

**Reply by MiguelNdeCarvalho** [reply]
Hey, the ls part can be always the same.

**Comment by viceice** [comment, marked as answer]
```json
   {
      "description": "use custom regex versioning",
      "matchPackagePatterns": ["(^|/)linuxserver\\/ddclient$"],
      "matchPackageNames": ["ddclient"],
      "versioning": "regex:^v(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)(?<compatibility>-ls)(?<build>\\d+)$"
    },
```

**Reply by viceice** [reply]
Oh, i see you need 5 version parts, which isn't supported.

**Reply by MiguelNdeCarvalho** [reply]
I think I can live with the ls part always the same!

**Reply by rarkins** [reply]
In that case you can treat the `-lsxxx` part as `compatibility` but that means any tags which don't match it will be ignored and not upgraded.

**Reply by MiguelNdeCarvalho** [reply]
Yap I have tried to do that, but I couldn't get it working. Do you know where I need to fix my regex? Thanks

**Reply by viceice** [reply]
You should ignore the last number, as it will change and probably never the same on different versions, so only capture the `-ls` part as compabillity

```
regex:^(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)\\.(?<build>\\d+)(?<compatibility>-ls)\\d+$
```

**Reply by MiguelNdeCarvalho** [reply]
Really thanks, it worked!
